Sources/TuistKit/Generator/GeneratorFactory.swift

```diff
@@ -95,8 +95,8 @@ public class GeneratorFactory: GeneratorFactorying {
         let graphMappers = graphMapperFactory.automation(
             config: config,
             testPlan: testPlan,
-            includedTargets: includedTargets,
-            excludedTargets: excludedTargets
+            includedTargets: Set(includedTargets.map(TargetPattern.pattern)),
+            excludedTargets: Set(excludedTargets.map(TargetPattern.pattern))
```

Comment:
What about adding a separate options `tuist generate --tagged XXX` so that it's more discoverable and less magic?

Reply:
Would you prefer to do `--tagged Framework1 Framework2` instead of `tag:Framework1 tag:Framework2`, or in addition?

Reply:
I would do it instead of

Reply:
I'd keep it as it's proposed since developers might have gotten familiar with the argument-based approach for specifying the targets to focus on. We can treat the arguments as "queries", which we can make very powerful. This is [an example](https://docs.vlt.sh/cli/commands/query) I recently came across with.